MAUI/Calendar/migration.md

```diff
@@ -279,17 +279,17 @@ this.Content = calendar;
 <tr>
 <td>{{'[BlackoutDatesViewMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_BlackoutDatesViewMode)'| markdownify }}</td>
 <td>Nil</td>
-<td>Not Supported.</td>
+<td>Use the [SelectableDayPredicate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_SelectableDayPredicate) for this requirement.</td>
 </tr>
 <tr>
 <td>{{'[CustomDayLabels](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_CustomDayLabels)'| markdownify }}</td>
 <td>Nil</td>
-<td>Not Supported.</td>
+<td>By using custom header (hides default view header using Height(https://help.syncfusion.com/maui/calendar/headers#customize-view-header-height) value as 0) in sample level you can customize.</td>
```

Comment:
<td>By using custom header (hides default view header using Height(https://help.syncfusion.com/maui/calendar/headers#customize-view-header-height) value as 0) in sample level, you can customize.</td>